deploy/package-operator/operator/kustomization.yaml

```diff
@@ -3,7 +3,7 @@ kind: Kustomization
 namespace: observability-operator
 images:
 - name: observability-operator
-  newTag: 0.0.17
```

Comment:
when the version gets bumped to 0.0.21, would we have to do this manually? If so, can we automate this somehow in Makefile?

Reply:
updated